From: Caleb Connolly <caleb.connolly@linaro.org>
Date: Mon, 28 Nov 2022 23:41:23 +0000
Subject: arm64: dts: qcom: c630: add debug uart

The c630 exposes its debug UART via a connector on the PCB, enable it.

Signed-off-by: Caleb Connolly <caleb.connolly@linaro.org>
---
 arch/arm64/boot/dts/qcom/sdm850-lenovo-yoga-c630.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/sdm850-lenovo-yoga-c630.dts b/arch/arm64/boot/dts/qcom/sdm850-lenovo-yoga-c630.dts
index 93996d4..6cac5b3 100644
--- a/arch/arm64/boot/dts/qcom/sdm850-lenovo-yoga-c630.dts
+++ b/arch/arm64/boot/dts/qcom/sdm850-lenovo-yoga-c630.dts
@@ -33,6 +33,7 @@ / {
 	chassis-type = "convertible";
 
 	aliases {
+		serial0 = &uart9;
 		hsuart0 = &uart6;
 	};
 
@@ -708,6 +709,10 @@ bluetooth {
 	};
 };
 
+&uart9 {
+	status = "okay";
+};
+
 &ufs_mem_hc {
 	status = "okay";
 
